packages/app-mobile/components/screens/ConfigScreen/SettingComponent.tsx

```diff
@@ -135,7 +135,7 @@ const SettingComponent: React.FunctionComponent<Props> = props => {
 						autoCorrect={false}
 						autoComplete="off"
 						selectionColor={theme.textSelectionColor}
-						keyboardAppearance={theme.settingKeyboardAppearance}
+						keyboardAppearance={theme.keyboardAppearance}
```

Comment:
`settingKeyboardAppearance` doesn't seem to be referenced or defined anywhere else.